package.json

```diff
@@ -6,18 +6,16 @@
     "node": "8.12.0"
   },
   "scripts": {
-    "build": "yarn run build-client && yarn run build-server && yarn run copy-reactapp",
+    "build": "lerna clean -y && lerna bootstrap && lerna run build && yarn run copy-reactapp",
```

Comment:
`lerna bootstrap` is very similar to `yarn link`, but a lot easier. It links local package dependencies within the `monorepo`